projects/js-packages/publicize-components/src/components/share-post/index.js

```diff
@@ -33,7 +33,7 @@ function showErrorNotice( message = __( 'Unable to share the Post', 'jetpack' )
  */
 function showSuccessNotice() {
 	const { createSuccessNotice } = dispatch( noticesStore );
-	createSuccessNotice( __( 'Post shared', 'jetpack' ), {
+	createSuccessNotice( __( 'Your post will be shared on the selected accounts.', 'jetpack' ), {
```

Comment:
```suggestion
	createSuccessNotice( __( 'Your post will be shared in a few seconds.', 'jetpack' ), {
```

Reply:
What about "Your post will be shared soon."?